front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarPhasesItem.tsx

```diff
@@ -36,20 +44,39 @@ const FilterSidebarPhasesItem = ({
     }),
   });
 
-  const disabled = !canContainIdeas(phase);
-
   return (
     <div ref={drop}>
-      <Menu.Item
-        active={active || (isOver && canDrop)}
-        onClick={onClick}
-        disabled={disabled}
```

Comment:
disabled was not necessary here/rather confusing. If you now click a phase that "can't contain ideas", it'll just show   "no results":

<img width="1006" alt="image" src="https://github.com/user-attachments/assets/c5920b6e-cce7-43dc-b8cb-0225ea603795" />

IMO, adding "disabled styles" to the label wasn't worth the effort.